File: test/leetcode/sliding_window/minimum-window-substring.test.ts
import { minWindow } from '../../src/sliding-window/minimum-window-substring';

describe('minimum window substring', () => {
  test('minWindow - test case 1', async () => {
    expect(minWindow('ADOBECODEBANC', 'ABC')).toStrictEqual('BANC');
  });
});

File: test/leetcode/sliding_window/minimum_window_substring_test.py
from leetcode.sliding_window.minimum_window_substring import Solution


soln = Solution()


def test_case_1():
    assert soln.minWindow("ADOBECODEBANC", "ABC") == "BANC"


def test_case_2():
    assert soln.minWindow("a", "a") == "a"
